src/components/AdminFilmCard/style.scss

```diff
@@ -0,0 +1,9 @@
+.filmsList .filmCard__button {
+    width: 80% !important;
```

Comment:
why using !important here? you should use it only if it is absolutely necessary and there's no other way